exercises/matthew_bowles/ex02_oo_basics/inc/Rectangle.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "Shape.h"
+
+class Rectangle :
+	public Shape
+{
+public:
+	Rectangle(double side1, double side2);
+	~Rectangle();
+
+	virtual double calcArea() const override;
```

Comment:
`virtual` keyword is not required here. However using `override` is a good idea :+1:  (Look out for this in other files)